app/components/doi-related-identifier.js

```diff
@@ -0,0 +1,213 @@
+/* eslint-disable no-useless-escape */
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+
+const relationTypeList = [
```

Comment:
Do we want to use the order from the metadata schema documentation? Or do this alphabetically?

Reply:
Sorting alphabetically will put all the `IsSomethingSomethingOf` together.  I am not sure of the benefit of using alphabetically on this specific case. The schema order seems fine as is paired relations. Maybe what I can do is to reverse the pairs in the schema order, so the active type goes first: 

  'Continues',
  'IsContinuedBy',

as opposed as in the schema:

  'IsContinuedBy',
  'Continues',

Reply:
You could sort the pairs alphabetically. I don't see any reasoning in the current sort order other than historical (newest relation types last).